README.md

```diff
@@ -3,406 +3,224 @@
-
-[Test API](https://devexpress.github.io/testcafe/documentation/test-api/index.html) consists of over two dozen methods that can emulate all actions one could possibly do with a webpage.
-Chained syntax allows for code that is easy to write and read.
-
-```js
-fixture `Example page`
-    .page `https://devexpress.github.io/testcafe/example`;
-
-test('Emulate user actions and perform a verification', async t => {
-    await t
-        .setNativeDialogHandler(() => true)
-        .click('#populate')
-        .click('#submit-button');
-
-    const location = await t.eval(() => window.location);
-
-    await t.expect(location.pathname).eql('/testcafe/example/thank-you.html');
-});
-```
-
-By using TypeScript to write your TestCafe tests, you get the advantages of strongly-typed languages such as: rich coding assistance, painless scalability, check-as-you-type code verification, and much more.
-
-To make debugging easier, TestCafe automatically generates source maps.
-To debug your test code, start a debugging session in an IDE that supports source maps.
+**Stable tests and no manual timeouts**<br/>
+TestCafe provides built-in assertions which wait until page elements load.
+You can change the maximum wait time.
+If elements load faster, tests skip the timeout and continue.
 
-### Flexible Selector System
+**Latest JS and TypeScript support**<br/>
+TestCafe supports the latest JavaScript features, including ES6, ES7 (for example, async/await).
+You can also [use TypeScript](https://devexpress.github.io/testcafe/documentation/test-api/typescript-support.html)
+if you prefer a strongly typed language.
 
-TestCafe supports a flexible selector system that provides API with rich capabilities for writing test scripts.
-You can access webpage elements in different ways using [selectors](https://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors.html). For example, you can use one of the following selector types:
+**Detects JS errors in your code**<br/>
+TestCafe reports JS errors that it finds on the webpage.
+Tests automatically fail because of that.
+However, you can disable this.
 
-* CSS selector
-* Text
-* DOM hierarchy
-* or use any custom logic
+**Concurrent tests launch**<br/>
+TestCafe can open multiple instances of the same browser to run parallel
+tests which decrease test execution time.
 
-The selector API provides methods that can be combined together, thus providing you with a flexible functional-style selector mechanism.
+**PageObject pattern support**<br/>
+The TestCafe's [Test API](https://devexpress.github.io/testcafe/documentation/test-api/)
+includes a high-level selector library, assertions, etc.
+You can combine them to implement readable tests with the [PageObject pattern](https://martinfowler.com/bliki/PageObject.html).
```

Comment:
We have our own recipe about this, I suppose it's better to add a link to it? https://devexpress.github.io/testcafe/documentation/recipes/using-page-model.html

Reply:
Yes